internal_worker_base.go

```diff
@@ -169,10 +175,14 @@ func (bw *baseWorker) execute(routineID int) {
 				if err == nil {
 					bw.retrier.Succeeded()
 				} else if isClientSideError(err) {
-					bw.logger.Info("Poll and processing failed with client side error", zap.Int(tagRoutineID, routineID), zap.Error(err))
+					traceLog(func() {
```

Comment:
this should not be trace level, it indicate something is wrong that probably needs to be looked at.